### PR TITLE
feat(tokens): update tokens

### PR DESCRIPTION
**Pull request by rzpcibot**
This PR was opened by the Token Upload GitHub action. It updates source token files based on the payload from Figma Plugin.

### CODE REVIEW

File: packages/blade/src/tokens/theme/bankingTheme.ts [hunk 1]
```diff
@@ -1067,12 +1067,12 @@ const colors: ColorsWithModes = {
       },
       border: {
         normal: {
-          lowContrast: globalColors.neutral.navyGrayDark[600],
-          highContrast: globalColors.neutral.navyGrayDark[600],
+          lowContrast: globalColors.neutral.navyGrayDark.a100,
```
Comment (kamleshchandnani) [inline packages/blade/src/tokens/theme/bankingTheme.ts:1070]: @saurav12 this was already merged right? why is this change coming again? is this new update? cc @chaitanyadeorukhkar

Comment (saurav12) [reply packages/blade/src/tokens/theme/bankingTheme.ts:1070]: @kamleshchandnani No no, I've changed it yesterday only.

Comment (chaitanyadeorukhkar) [reply packages/blade/src/tokens/theme/bankingTheme.ts:1070]: Looks like another change

Comment (saurav12) [reply packages/blade/src/tokens/theme/bankingTheme.ts:1070]: Wait! Do not merge this

Comment (saurav12) [reply packages/blade/src/tokens/theme/bankingTheme.ts:1070]: All good, this can be merged!

**Pull request merged**